x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_custom_source/configure_custom.tsx

```diff
@@ -53,14 +53,19 @@ export const ConfigureCustom: React.FC<ConfigureCustomProps> = ({ sourceData })
 
   const {
     serviceType,
+    baseServiceType,
     configuration: { documentationUrl, githubRepository },
     name,
     categories = [],
   } = sourceData;
 
   return (
     <>
-      <AddSourceHeader name={name} serviceType={serviceType} categories={categories} />
+      <AddSourceHeader
+        name={name}
+        serviceType={baseServiceType || serviceType}
```

Comment:
suggestion: Use nullish coalescing (`??`)

Reply:
d'oh! thank you for reminder